keyboards/falsonix/fx19/keyboard.json

```
@@ -0,0 +1,68 @@
+{
+    "manufacturer": "Falsonix",
+    "keyboard_name": "falsonix/fx19",
+    "maintainer": "falsonix",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 5,
+        "animations": {
+            "rainbow_mood": true,
+            "rainbow_swirl": true
+        },
+        "default": {
+            "animation": "rainbow_swirl"
+        }
+    },
+    "ws2812": {
+        "pin": "F7"
+    },
+    "matrix_pins": {
+        "cols": ["C7", "C6", "B6", "B5"],
+        "rows": ["F0", "F1", "F4", "F5", "F6"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0x4658"
+    },
+    "layouts": {
+        "LAYOUT_numpad_4x5": {
```

This would usually be `5x4` with the exception of the bottom row differences. As this doesnt match a community layout, its going to be best to just do....
```suggestion
        "LAYOUT": {
```